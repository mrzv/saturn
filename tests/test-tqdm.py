from tqdm import tqdm
import time

fmt = '{l_bar}{bar}| {n_fmt}/{total_fmt}'       # to avoid diff errors

#---#

for i in tqdm(range(20), bar_format=fmt):
    time.sleep(.1)

#---#


for i in tqdm(range(10), bar_format=fmt):
    for j in tqdm(range(30), bar_format=fmt, leave = False):
        time.sleep(.05)
