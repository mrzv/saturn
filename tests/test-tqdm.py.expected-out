>>> from tqdm import tqdm                                                       
... import time                                                                 
...                                                                             
... for i in tqdm(range(20), bar_format='{l_bar}{bar}| {n_fmt}/{total_fmt}'):   
...     time.sleep(.1)                                                          

  0%|          | 0/20  5%|▌         | 1/20 10%|█         | 2/20 15%|█▌        | 3/20 20%|██        | 4/20 25%|██▌       | 5/20 30%|███       | 6/20 35%|███▌      | 7/20 40%|████      | 8/20 45%|████▌     | 9/20 50%|█████     | 10/20 55%|█████▌    | 11/20 60%|██████    | 12/20 65%|██████▌   | 13/20 70%|███████   | 14/20 75%|███████▌  | 15/20 80%|████████  | 16/20 85%|████████▌ | 17/20 90%|█████████ | 18/20 95%|█████████▌| 19/20100%|██████████| 20/20100%|██████████| 20/20

