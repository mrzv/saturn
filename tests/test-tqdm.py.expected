from tqdm import tqdm
import time

for i in tqdm(range(20), bar_format='{l_bar}{bar}| {n_fmt}/{total_fmt}'):
    time.sleep(.1)
#o> 100%|██████████| 20/20
